test/integration/config/esm-bad/index.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```

Comment:
so i found in my PR if you rename these to .mjs you'll get passing linting.  However you'll need to edit your test/.eslintrc.js to add the following: https://github.com/newrelic/node-newrelic/pull/1349/files#diff-840ad34d5df7045c9d82c26b46b7eb1a16af1602b1fa393590fc548518a1e7c9

Reply:
Didn't need that change, changing the extension to `.mjs` used the proper linting rules.